src/components/app.tsx

```diff
@@ -774,38 +804,155 @@ export class App extends React.Component<AppProps, AppState> {
       }
     }
 
-    renderConnectToDeviceButton() {
-      const { sensorManager } = this.state;
-      // Check if this sensorManger supports device connection
-      if (isConnectableSensorManager(sensorManager) && this.props.sensorManager) {
-        if (sensorManager.deviceConnected) {
-          return  <Button className="connect-to-device-button" onClick={this.disconnectFromDevice} >
-                    Disconnect from Device
-                  </Button>;
+    zeroSensor = (slotNum: number) => () => {
+        let { sensorSlots } = this.state;
+        if (sensorSlots[slotNum].sensor) {
+            sensorSlots[slotNum].sensor.zeroSensor();
+            this.setState({ sensorSlots });
+        }
+    }
+
+    renderStatusMessage() {
+        const { sensorManager, sensorConfig } = this.state;
+        let wirelessIconClass = "wireless-status-icon ";
+        if (sensorManager!==null) {
```

Comment:
```suggestion
        if (sensorManager != null) {
```
in case `sensorManager === undefined`.

Reply:
Fixed.